functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/MavenExtensionRegisterer.java

```diff
@@ -36,4 +42,37 @@ void addSmartTestingExtension(String version) {
         }
     }
 
+    void addSmartTestingShadedExtensionIntoLibExtFolder(String stVersion, String maven3Version) {
```

Comment:
I was wondering if it would make sense to have 2 classes here - one for installing extension using the new way and other for old-school way? In addition maybe slicing this method body to smaller methods named in the way that it's clear what are the steps involved could be useful?

Reply:
Personally, I would keep it inside of one class - it makes sense for me as both cases is just registering the ST extension, which is pretty good explained by the class name: `MavenExtensionRegisterer`. Are you fine with it? If not, then which names do you propose for the classes in case it was separated.
But yeah, I can slice the method to smaller pieces 👍

Reply:
Ok, I just got to this: https://github.com/arquillian/smart-testing/pull/209/files#r143059742
The names would be:
* `MavenExtensionFileRegisterer`
* `MavenExtensionShadedJarRegisterer`